src/network/http_server.rs

```diff
@@ -297,18 +297,17 @@ pub struct CastsByParentRequest {
 #[derive(Debug, Clone, Deserialize, Serialize)]
 pub struct ReactionRequest {
     fid: u64,
-    #[serde(rename = "reactionType")]
-    reaction_type: ReactionType,
-    #[serde(rename = "targetCastId", skip_serializing_if = "Option::is_none")]
+    #[serde(rename = "target_fid", skip_serializing_if = "Option::is_none")]
     target_cast_id: Option<CastId>,
```

Comment:
If you're rename this target_fid, rename the field to match and have the data type be u64